app/src/main/java/com/example/evently/ui/auth/AuthActivity.java

```diff
@@ -97,19 +101,84 @@ private void showRegisterForm() {
         getSupportFragmentManager()
                 .setFragmentResultListener(
                         RegisterFragment.resultKey, this, (var key, var bundle) -> {
-                            // TODO (chase): Persist the data in the bundle to register user.
-                            // NOTE: If the bundle contains an email that already exists in the DB,
-                            // show an error and prompt the user to login instead of registering.
-                            startActivity(transition);
-                            finish();
+                            String email = bundle.getString("email");
```

Comment:
Let's put this whole thing in a separate method. This is getting too nested down below.